nbdime/webapp/templates/compare.html

```diff
@@ -9,18 +9,18 @@ <h3>Notebook Diff/Merge</h3>
       <form id="nbdime-compare-form" class="nbdime-forms">
         <fieldset>
           <legend>Please input filenames of notebooks to compare:</legend>
-          <label>Local:</label>
-          <input id="compare-local" type="text" name="local" value="{{ config_data['local']|e }}" />
-          <label>Base:</label>
-          <input id="compare-base" type="text" name="base" value="{{ config_data['base']|e }}" />
-          <label>Remote:</label>
-          <input id="compare-remote" type="text" name="remote" value="{{ config_data['remote']|e }}" />
+          <label>Local:
```

Comment:
Keeping atomic labels with `for="id"`  might make it easier to translate in the #709 but for the sake of this one it does not matter much.